Practice#1

```diff
@@ -1,4 +1,3 @@
-Implement your own list. It should work with all collection api and with all methods of List.
 Class CustomList
```

Comment:
to implement your own list you need to extend AbstractList or implement List interface

Reply:
I got it, Kseniya . I will modify it right away.